artifacts/deploy/webhook-configuration.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: mutating-config
+  name: karmada-mutations
   labels:
-    app: mutating-config
```

Comment:
Can we modify the labels first?

Based on the description in the issue, the user is unable to distinguish which project the webhook belongs to. If we already have a keyword like `karmada` in the labels, I think the user would be able to distinguish them.

Reply:
![image](https://github.com/user-attachments/assets/a94170ae-2131-4382-a205-390d0b48950f)

I checked the `Validatingwebhookconfigurations` in one of my clusters. They can indeed be distinguished by their names which project they belong to. 😂

Reply:
https://github.com/karmada-io/karmada/pull/5246